meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Resin Hostname Configuration
+Requires=resin-device-uuid.service
+After=resin-device-uuid.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=@BASE_BINDIR@/bash @BINDIR@/%p
```

Comment:
The script starts with `#!/bin/sh` and is also installed as executable. It should be plain `ExecStart=@BINDIR@/%p`